extension/data/deinflect.dat

```diff
@@ -333,4 +336,34 @@ imperative negative
 り	る	640	24
 れ	る	640	23
 れ	れる	384	24
-ろ	る	384	23
\ No newline at end of file
+ろ	る	384	23
+です	だ	8320	13
+だった	だ	8320	14
+でした	だ	8320	6
+ではない	だ	8320	15
+じゃない	だ	8320	15
+ではないです	だ	8320	1
```

Comment:
I noticed that rikaikun doens't usually include です ending for polite negative (熱くないです for example) since I guess people should know that the です is connected.

That being said, this seems useful for new learners so I think it's fine to keep (and maybe think of extending this logic to other adjective types later)

Reply:
I added an entry so that です is considered a polite inflection for i-adjectives. This does not work when the adjectives are in past form, however, which I believe is due to a different type mask. So, I added another separate entry instead.